src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessarysemicolonafteroutertypedeclaration/Example2.java

```diff
@@ -8,20 +8,23 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.coding.unnecessarysemicolonafteroutertypedeclaration;
+
 // xdoc section -- start
-class A {
+class Example2 {
 
 }; // violation
```

Comment:
add nested class here to make the structure of both examples is exactly the same
rule no. 1 in https://github.com/checkstyle/checkstyle/issues/13345#issue-1786334846

Reply:
Done.